.github/workflows/license_tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Run License Tests
```

Comment:
This can be replaced with https://github.com/NeonGeckoCom/neon-stt-plugin-google_cloud_streaming/blob/dev/.github/workflows/license_tests.yml